src/server/tasks/ngp-van/fetch-van-activist-codes.ts

```diff
@@ -24,26 +27,29 @@ export interface VANActivistCode {
 
 export const fetchVANActivistCodes: Task = async (
   payload: GetActivistCodesPayload,
-  _helpers: any
+  helpers
```

Comment:
Suggestion: Add type `JobHelpers`

Reply:
The `_` and `any` part is being removed, and helpers is now being used in there. My suggestion was to only add the type, helps IDEs. I think my suggestion was confusing, sorry about that.